tox.ini

```diff
@@ -22,9 +29,12 @@ commands =
     pip install -e .
     py.test {posargs}
 
+[testenv:py{36,37,38,39}-djangomaster]
+ignore_errors = True
+
 [testenv:flake8]
 basepython =
-    python3.7
```

Comment:
Can we skip the install for this environment? It saves a bit of time from installing the package (and Django and it's dependencies too)

https://tox.readthedocs.io/en/latest/config.html#conf-skip_install

Reply:
Good idea. `skip_install` added.